backend/api/models.py

```
@@ -29,6 +29,7 @@ class User(AbstractBaseUser, PermissionsMixin):
     
     firebase_uid = models.CharField(max_length=128, unique=True)
     email = models.EmailField(unique=True)
+    phone_number = models.CharField(max_length=20, blank=True, null=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)